go/v1/resource_descriptor_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Tests for in-toto attestation ResourceDescriptor protos.
+ */
+
+package v1
+
+import (
+	"testing"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/structpb"
+	"github.com/stretchr/testify/assert"
```

Comment:
can you run `go mod tidy`? I'm surprised this wasn't added to `go.mod` and `go.sum`.

Reply:
```
go test ./...
# github.com/in-toto/attestation/go/v1
go/v1/resource_descriptor_test.go:13:2: no required module provides package github.com/stretchr/testify/assert; to add it:
        go get github.com/stretchr/testify/assert
FAIL    github.com/in-toto/attestation/go/v1 [setup failed]
?       github.com/in-toto/attestation/examples/go      [no test files]
?       github.com/in-toto/attestation/go/predicates/link/v0    [no test files]
?       github.com/in-toto/attestation/go/predicates/vsa/v0     [no test files]
FAIL
make: *** [Makefile:13: go_test] Error 1
```

Reply:
I suggest adding a github workflow that runs `make go_test` as well.

Reply:
Ah, yes, I'll push the updated go.mod and go.sum.

Reply:
Done.